simulator/Vagrantfile

```diff
@@ -110,8 +110,9 @@ Vagrant.configure("2") do |config|
     config.vm.provision "file", source: "native_streaming.service", destination: "/tmp/native_streaming.service"
     config.vm.provision "shell", inline: "cp /tmp/*.service /etc/avahi/services/"
 
-    # Copy opendaq-config.json file 
-    config.vm.provision "file", source: "opendaq-config.json", destination: "/tmp/opendaq-config.json"
+    # Copy opendaq-config.json file
+    config.vm.provision "file", source: "opendaq-config.json", destination: "/tmp/"
+    config.vm.provision "shell", inline: "cp /tmp/opendaq-config.json /home/opendaq/opendaq/"
```

Comment:
Have you tested the simulator image built with these changes - does it provide the ref_device as the root?

Reply:
@denise-opendaq